apps/nextjs/src/components/board/items/item-actions.tsx

```diff
@@ -128,8 +127,8 @@ export const useItemActions = () => {
                 if (item.id !== itemId) return item;
                 return {
                   ...item,
-                  ...("integrations" in item
-                    ? { integrations: newIntegrations }
+                  ...("integrationIds" in item
+                    ? { integrationIds: newIntegrations }
                     : {}),
                 };
```

Comment:
Recommendation: use produce from immer for this kind of operations

Reply:
Immer looks great and all but it seems like it breaks some things and I don't want to let it throw errors when I don't know where they are thrown. The syntax would be great